src/AzureRenderManager/WebApp/Startup.cs

```diff
@@ -221,16 +221,8 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, Microsoft.AspNetCore.Hosting.IHostingEnvironment env)
         {
-            if (env.IsDevelopment())
-            {
-                app.UseDeveloperExceptionPage();
-            }
-            else
-            {
-                app.UseExceptionHandler("/Home/Error");
-                app.UseHsts();
-            }
-
+            app.UseExceptionHandler("/Environments/Error");
+            app.UseHsts();
```

Comment:
if we're turning on HSTS we should disable HTTP in the web application too

Reply:
We have the HTTP->HTTPS redirect enabled too.